### ISSUE TITLE
Do not use the aws_main_route_table_association & aws_default_route_table together.

### ISSUE THREAD

**Issue by willejs** [terraform-aws-modules/terraform-aws-vpc#95]
As per the warning in the docs.

https://www.terraform.io/docs/providers/aws/r/default_route_table.html

**Comment by antonbabenko** [comment]
Yes, this is a good note, but `route` is not specified inline in `aws_default_route_table` resource in this module, so it should not be a problem for a conflict.

Am I missing something?

**Comment by willejs** [comment]
yeah, it will destroy the routes in the default routing table, unless you define and import them.
```
The aws_default_route_table behaves differently from normal resources, in that Terraform does not create this resource, but instead attempts to "adopt" it into management. We can do this because each VPC created has a Default Route Table that cannot be destroyed, and is created with a single route.

When Terraform first adopts the Default Route Table, it immediately removes all defined routes. It then proceeds to create any routes specified in the configuration. This step is required so that only the routes specified in the configuration present in the Default Route Table.
```

**Comment by antonbabenko** [comment]
**value** vs **problems** suggests me to abandon the idea of managing them in this module completely. Will you be able to submit a PR?

**Comment by antonbabenko** [comment]
Resolved as part of #111.
